src/Our.Umbraco.Ditto/ComponentModel/Attributes/UmbracoPropertyProcessorAttribute.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace Our.Umbraco.Ditto
+{
+    /// <summary>
+    /// The Umbraco property processor attribute.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property)]
+    public class UmbracoPropertyProcessorAttribute : DittoProcessorAttribute
```

Comment:
Yup, totally fine. I've removed the Ditto prefix from all aswell, and renamed the Picker one to be UmbracoPicker.